package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "keypair_lib",
```

Comment:
You probably want to refactor this to keypairoom?

Reply:
it depends, let's discuss about it at 16 :)

Reply:
name changed to keypairoom